localization/cs_CZ.inc

```diff
@@ -5,8 +5,8 @@ $labels = array();
 $labels['buttonjunk'] = 'Oznacit jako spam';
 $labels['markasjunk'] = 'Jako spam';
 $labels['buttonnotjunk'] = 'Tato zpráva není spam';
-$labels['markasnotjunk'] = 'Nemí spam';
+$labels['markasnotjunk'] = 'Není spam';
 
 $messages = array();
 $messages['reportedasjunk'] = 'Úspešne hlášeny jako spam';
-$messages['reportedasnotjunk'] = 'Úspešne hlášeny jako spam';
+$messages['reportedasnotjunk'] = 'Úspešne hlášeny jako spam';  // same message as for 'reportedasjunk'
```

Comment:
if the line is wrong then please either correct it or remove it. i dont think we need comments in locals files